pkg/pxc/app/pvc.go

```diff
@@ -24,5 +24,7 @@ func VolumeSpec(vspec *api.VolumeSpec) corev1.PersistentVolumeClaimSpec {
 		StorageClassName: vspec.PersistentVolumeClaim.StorageClassName,
 		AccessModes:      vspec.PersistentVolumeClaim.AccessModes,
 		Resources:        vspec.PersistentVolumeClaim.Resources,
+		DataSource:       vspec.PersistentVolumeClaim.DataSource,
```

Comment:
@ebuildy could you please add these fields to our template cr.yaml like it was done for e.g. storageClassName https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L179 and we will accept your PR. Thanks!

Reply:
Dobryj den’ @hors , hope you are doing well, thanks you for your review, I have just updated the cr.yaml file.